From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 9 May 2024 16:59:57 +0200
Subject: glib/tests/gutils-user-database: Add test dependency on preload
 library

We do preload the library but that's not set as test dependency and so
it may not be built

Origin: upstream, 2.80.3, commit:c6770f964c8489b7bec42ecf8be7a2175f821ca0
---
 glib/tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glib/tests/meson.build b/glib/tests/meson.build
index 92486cf..c448b81 100644
--- a/glib/tests/meson.build
+++ b/glib/tests/meson.build
@@ -263,7 +263,7 @@ else
 
     glib_tests += {
       'gutils-user-database' : {
-        'depends' : [],
+        'depends' : getpwuid_preload,
         'env' : {
           var_preload: getpwuid_preload.full_path()
         },
